src/helm/config/model_deployments.yaml

```diff
@@ -105,28 +105,69 @@ model_deployments:
 
 
   # Amazon
-  - name: amazon/titan-text-lite-v1
-    model_name: amazon/titan-text-lite-v1
+  # Titan on Amazon Bedrock
+
+  - name: amazon/amazon.titan-text-lite-v1
+    model_name: amazon/amazon.titan-text-lite-v1
     tokenizer_name: huggingface/gpt2
     max_sequence_length: 4000
     client_spec:
       class_name: "helm.clients.bedrock_client.BedrockTitanClient"
 
-  - name: amazon/titan-tg1-large
-    model_name: amazon/titan-tg1-large
-    tokenizer_name: huggingface/gpt2
+  - name: amazon/amazon.titan-text-express-v1
+    model_name: amazon/amazon.titan-text-express-v1
```

Comment:
Make this `model_name: amazon/titan-text-express-v1` to follow our model naming convention.